app/views/stores/show.html.erb

```diff
@@ -1,4 +1,4 @@
-<h1> This is the store show <h1>
+<h1> This is the store show page for <%=@store.name%><h1>
 
```

Comment:
I believe Saralis will be editing this page. If you'd like to take it on, let's switch out the trello responsibilities.